src/main/java/com/employeesService/EmployeesService/repository/DepartmentRepository.java

```diff
@@ -13,6 +14,10 @@
 
     Optional<Department> findById(String deptNo);
 
+    Optional<Department> findByIdWithEmployees(String deptNo);
+
+    Optional<DepartmentEmployee> findDepartmentWithEmployees(String deptNo, Long empNo);
```

Comment:
This is not used, right? Also doesn't belong here and it's not well defined, so please remove it and its implementation

Reply:
They both get used, one is to find the department that calls the list of departments that have employees, the other is to find a DepartmentEmployee which is used at the delete method where the toDate is assigned

Reply:
Yes, but I meant it should not be used, so after this change, the second method is useless :)

https://github.com/TaulantFazliu/employees-service/pull/5#discussion_r750959571